src/methods.rs

```diff
@@ -49,9 +49,11 @@ pub mod execute {
     use super::*;
 
     /// Execute an Ethereum transaction.
-    pub fn tx(txn: &[u8], ctx: &mut TxnContext) -> Fallible<ExecutionResult> {
+    //pub fn tx(txn: &[u8], ctx: &mut TxnContext) -> Fallible<ExecutionResult> {
+    pub fn tx(call: &super::super::dispatcher::DecodedCall, ctx: &mut TxnContext) -> Fallible<ExecutionResult> {
```

Comment:
Consider importing `DecodeCall` and change the import to use `crate::` instead `super::super::`.